src/Microsoft.VisualStudio.Web.CodeGenerators.Mvc/Controller/ControllerWithContextGenerator.cs

```diff
@@ -17,6 +17,7 @@ namespace Microsoft.VisualStudio.Web.CodeGenerators.Mvc.Controller
 {
     public class ControllerWithContextGenerator : ControllerGeneratorBase
     {
+        private string AreaName = string.Empty;
```

Comment:
private names start with _ and lower case? or you don't have this convention in this repo?